2.6/Dockerfile.rhel8

```diff
@@ -43,7 +43,7 @@ RUN yum -y module enable ruby:$RUBY_VERSION && \
     redhat-rpm-config \
     " && \
     yum install -y --setopt=tsflags=nodocs ${INSTALL_PKGS} && \
-    yum -y clean all --enablerepo='*' && \
+    yum clean all -y && \
```

Comment:
The previous version is the correct one here. Likely did not get into downstream yet.

Reply:
Oh, ok.